bin/install_upgrade.escript

```diff
@@ -144,9 +144,10 @@ parse_arguments([VersionStr|Rest], Acc) ->
     parse_arguments(Rest, [{version, Version}] ++ Acc).
 
 unpack_release(RelName, TargetNode, Version) ->
+    StartScriptExists = filelib:is_dir(filename:join(["releases", Version, "start.boot"])),
```

Comment:
the changes in this file deserves a separate PR with a more focused description.

Reply:
I'll do this.

Reply:
https://github.com/emqx/emqx/pull/7165